src/vivarium/config_tree.py

```diff
@@ -192,7 +192,9 @@ def __repr__(self):
                                    for layer, value in self._values.items()]))
```

Comment:
the __str__ method looks good! I think we have the same ordering problem in the __repr__ though right? Because we're using the order in the values dict instead of the layers list